crates/ruff_python_parser/src/error.rs

```diff
@@ -457,6 +458,7 @@ impl Display for UnsupportedSyntaxError {
             UnsupportedSyntaxErrorKind::Match => "`match` statement",
             UnsupportedSyntaxErrorKind::Walrus => "named assignment expression (`:=`)",
             UnsupportedSyntaxErrorKind::ExceptStar => "`except*`",
+            UnsupportedSyntaxErrorKind::PosOnly => "positional-only parameters",
```

Comment:
What about "positional-only parameter separator" as we're highlighting the `/` token in the source code? The word "separator" has been used in the [PEP](https://peps.python.org/pep-0570/#origin-of-as-a-separator).